integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -670,11 +654,9 @@ public int rename(String oldPath, String newPath) {
         return -ErrorCodes.EEXIST();
       }
       mFileSystem.rename(oldUri, newUri);
-      synchronized (mOpenFiles) {
-        if (mOpenFiles.contains(PATH_INDEX, oldPath)) {
-          OpenFileEntry oe = mOpenFiles.getFirstByField(PATH_INDEX, oldPath);
-          oe.setPath(newPath);
-        }
+      if (mOpenFiles.contains(PATH_INDEX, oldPath)) {
+        OpenFileEntry oe = mOpenFiles.getFirstByField(PATH_INDEX, oldPath);
+        oe.setPath(newPath);
```

Comment:
If we don't synchronize here, it is possible that `mOpenFiles.contains(PATH_INDEX, oldPath)` is no longer true

Reply:
Make sense. Will add synchronization back, thanks